tests/neg/i2673.scala

```diff
@@ -0,0 +1,4 @@
+package Foos //error
```

Comment:
This should fo into `neg-custom-args` and be compiled with `-Xfatal-warnings`. Here is an example https://github.com/lampepfl/dotty/blob/78bd3dd8ce5822171ccf252864e2e5c3d76747a4/compiler/test/dotty/tools/dotc/CompilationTests.scala#L197